Djikstra/Djikstra/main.cpp

```diff
@@ -3,11 +3,76 @@
 #include <string>
 #include <fstream>
 #include <vector>
+#include <map>
+
+#include "Headers.h"
 
-using namespace std;
 
 int main()
 {
+	std::string name1, name2;
+	float dist=0;
+	Graph::Graph graph;
+	Graph::Vertex node;
+	bool end1=0, end2=0;
+
+
+	// Loading data
+	std::fstream file_data;
+	file_data.open("distances.txt", std::ios::in);
+
+
+	if (file_data.good() == false)
+	{
+		std::cout << "Error with oppening";
+		exit(1);
+	}
+
+
+	while (!file_data.eof())
+	{
+		file_data >> name1 >> name2 >> dist;
+
+		if (!graph.node_exist(name1 ))
+		{
+			graph.add_node(name1 );
+		}
+		if (!graph.node_exist(name2 ))
+		{
+			graph.add_node(name2 );
+		}
+		graph.add_edge(name1, name2, dist );
+	}
+
+	file_data.close();
+	// data loaded
+
+
+	// searching shortest way from name1 to name2
+	std::cout << "From where, to where?" << std::endl;
```

Comment:
This should be done via [command line arguments](https://www.geeksforgeeks.org/command-line-arguments-in-c-cpp/). You can set these in your environment (read [here](https://stackoverflow.com/a/298713)).